tools/pipelines/test-real-service.yml

```diff
@@ -128,7 +129,7 @@ jobs:
     - task: Npm@1
       displayName: '[end-to-end tests] npm run test:realsvc:routerlicious:report'
       continueOnError: true
-      timeoutInMinutes: 15
```

Comment:
these finish in 15, might just want to keep them contrained

Reply:
I think I'd rather give it some breathing room